images/ceph/Makefile

```diff
@@ -124,15 +124,15 @@ csv: $(OPERATOR_SDK) $(YQ) ## Generate a CSV file for OLM.
 csv-clean: $(OPERATOR_SDK) $(YQ) ## Remove existing OLM files.
 	@rm -fr ../../cluster/olm/ceph/deploy/* ../../cluster/olm/ceph/templates/*
 
-# list-image creates list of images for offline installation
-list-image:
-	@echo "producing list of images for offline installation";\
-	# remove the file if already exists
-	rm -f $(MANIFESTS_DIR)/images.txt;\
```

Comment:
For @subhamkrai, note that in make targets, it is only necessary to specifically end a command (`;`) and then ignore the newline (`\`) for complex scripts or in loops. For something like this, it's really unnecessary. It's the same to remove the file in its own subshell. I made this change throughout the target to simplify reading/writing it.

Reply:
So we don't need to use `;` in here and every command will run as a single command? As when I was reading it was mentioned that `;` was to end the command and I put `;` when a single command is over example `rm -f $(MANIFESTS_DIR)/images.txt;`.

Reply:
got it thanks